config/crds/autoscaling_v1alpha1_hvpa.yaml

```diff
@@ -170,6 +180,16 @@ spec:
                         type: object
                       type: array
                   type: object
+                updatePolicy:
```

Comment:
Same as above.

Reply:
Inspired from [VPA](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2/types.go#L91).
IMHO, we can keep it same, unless there is a strong reason to change it. WDYT?

Reply:
Please see above.